src/overlays/actors/ovl_Bg_Jya_Bombiwa/z_bg_jya_bombiwa.c

```diff
@@ -1,3 +1,8 @@
+/*
+ * File: z_bg_jya_bombiwa.c
+ * Overlay: ovl_Bg_Jya_Bombiwa
+ * Description: Spirit Temple top room bombable wall
+ */
```

Comment:
Pretty minor but there should be a break between the file comment and the includes, ideally
```suggestion
 */

```